src/delphi/eval/utils.py

```diff
@@ -25,3 +27,18 @@ def get_next_logprobs(
     logprobs = get_all_logprobs(model, input_ids[:, :-1])
     next_tokens = input_ids[:, 1:]
     return gather_logprobs(logprobs, next_tokens)
+
+
+def load_validation_dataset(dataset_name: str) -> Dataset:
+    if "/" not in dataset_name:
+        dataset_name = f"delphi-suite/{dataset_name}"
+    data_str = f"data/validation-*.parquet"
+    dataset = load_dataset(
+        dataset_name,
+        data_files=data_str,
+        verification_mode="no_checks",
+        # this seems to be the only split when using data_files
+        # regardless of the files we're actually loading
+        split="train",
+    )
+    return cast(Dataset, dataset)
```

Comment:
One question: we are still add a BOS token to every sequence in `tokenized = load_validation_dataset("tinystories-v2-clean-tokenized-v0")` before batching, right?

Reply:
No, we don't prepend anything in the tokenized dataset. -v0 is weird and doesn't have BOS, but this is what the v0 models were trained on. Final version will have BOS